straxen/url_config.py

```diff
@@ -303,6 +540,20 @@ def format_arg(arg: str, **kwargs):
     return arg.format(**kwargs)
 
 
+@URLConfig.register_preprocessor('cmt')
+def replace_global_version(correction, version=''):
+    if isinstance(version, str) and version.startswith('global'):
+        local_versions = get_cmt_local_versions(version)
+        v = local_versions.get(correction, version)
+        return dict(version=v)
```

Comment:
You don't actually use this right? Apart from the tests? We would have to modify the `apply_cmt_version` function to modify global versions instead of local versions right?

Reply:
This is to replace the functionality of apply_cmt_version

Reply:
but you need to specify somewhere which global_vx you are using right?

Reply:
either is fine, in any case we can remove the logic in apply_cmt_version that evaluates the URL to get the cmt name.
If we go for option 1 then apply_cmt_version would just set the version keyword in all cmt URLs to the global version so it would be a one-liner